### DISCUSSION TITLE
file_exists(): File name is longer than the maximum allowed path length on this platform (4096):

### DISCUSSION THREAD

**Discussion by groxw** [laravel/framework#48589] [Q&A]
### Laravel Version

10.25

### PHP Version

8.1

### Description

Hello, Ive development laravel app, in localhost it works properly when i try to upload it into server, it turns out something like :file_exists(): File name is longer than the maximum allowed path length on this platform (4096):

How can i resolve this issue?

file_exists(): File name is longer than the maximum allowed path length on this platform (4096):

### Steps To Reproduce
-Mary UI - Livewire 3 - Laravel 10

**Comment by henzeb** [comment]
What is the path and / filename and what is its length?
